lib/stack_master/command.rb

```diff
@@ -36,9 +42,26 @@ def success?
     def error_message(e)
       msg = "#{e.class} #{e.message}"
       msg << "\n Caused by: #{e.cause.class} #{e.cause.message}" if e.cause
+      if defined?(@options)
```

Comment:
Now should we define an accessor method
```
def options
  @options ||= {}
end
```

So we don't have to check if it's defined?

Reply:
Added in 9621019.